projects/aca-shared/src/lib/components/open-in-app/open-in-app.component.scss

```diff
@@ -0,0 +1,15 @@
+.container{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+::ng-deep .mat-dialog-container{
```

Comment:
you don't need "ng-deep" here as long as View Encapsulation is set to "None" as we do in all components

Reply:
noted and resolved